site/source/docs/getting_started/test-suite.rst

```diff
@@ -181,11 +181,19 @@ As with all the test suites, you can also run a specific benchmark:
   # Run one specific benchmark
   test/runner benchmark.test_skinning
 
-Usually you will want to customize the in `test/test_benchmark.py` to
-run the benchmarks you want (there is currently no external config file). Things
-you may want to modify include:
+You can also specify which benchmarkers are run by using the environment
+variable `EMTEST_BENCHMARKERS`. It accepts a comma separated list of named
+benchmarkers (names can be found in `named_benchmarkers` in
+`test/test_benchmark.py`).
+
+.. code-block:: bash
+
+  # Run one specific benchmark and with clang and v8.
+  EMTEST_BENCHMARKERS=clang,v8 test/runner benchmark.test_skinning
+
+To further customize how the benchmarks are run, you will want to edit the file
+`test/test_benchmark.py`. Some of the options include:
```

Comment:
Sorry to be a pain but can we split this change into its own PR?

Could we also remove `EMTEST_FORCE64` at the same time?  (since I believe this feature superseded that).

Reply:
Split out into https://github.com/emscripten-core/emscripten/pull/22139